### DISCUSSION TITLE
Delta Pro not connecting

### DISCUSSION THREAD

**Discussion by bluefire100** [vwt12eh8/hassio-ecoflow#62] [Q&A] [answered]
I tried to connect my Delta Pro to HA with this integration. But after entering the IP address of my Delta Pro nothing more happens. It seems to search endless or not to find it, although the device is present. Any ideas? btw: which characters is the MAC address of Delta Pro beginning with?

**Comment by vwt12eh8** [comment, marked as answer]
Newly purchased device? 
Currently this integration does not work with the latest firmware. 
Please wait until a compatible version is developed.

See #58

**Reply by bluefire100** [reply]
Thx. Yes my Delta Pro is brand-new, just did firmware update to newest version. That explains. Let's wait for the new version.

**Reply by stefan-b-b** [reply]
is there a way to create a sensor using CURL -X GET API? Sorry I am struggling to get it. I can get the data in Json but then not into a sensor. Any idea?

**Reply by DarrylGNOLA** [reply]
@bluefire100 and @stefan-b-b

Ecoflow has released new wifi firmware that works with this integration.

**Reply by bluefire100** [reply]
I dont see new firmware in Ecoflow app

**Reply by DarrylGNOLA** [reply]
@bluefire100 Where are you located? I see others in the United States have updated, but a person from Germany posted he didn't see the update.

**Reply by bluefire100** [reply]
I'm running v1.0.1.18 / v3.0.1.11 (WLAN)

**Reply by arnieatgit** [reply]
I have the same problem (Austria), 1.0.1.18 / v3.0.1.11 (WLAN) and no Update available.

I contacted a couple of days ago the support, they answered me that's the current Version (1.0.1.18). I told them now about Wi-Fi Version, waiting for an reply ...

**Reply by bluefire100** [reply]
I'm using now https://github.com/tolwi/hassio-ecoflow-cloud which works fine for me and delivers all the wanted informations to me. 
Where in Ecoflow app can I find the IP address of my Delta Pro and Smart Home Panel? Couldn't find it so far.

**Reply by DarrylGNOLA** [reply]
@bluefire100 "Where in Ecoflow app can I find the IP address of my Delta Pro and Smart Home Panel?" I personally logged into my own Wifi Router and found them among my list of devices.
